config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-peribolos.yaml

```diff
@@ -13,7 +13,7 @@ postsubmits:
     cluster: k8s-infra-prow-build-trusted
     decorate: true
     decoration_config:
-      timeout: 180m
+      timeout: 210m
```

Comment:
Can you pls add a TODO comment saying this needs to be bumped down once we figure out what's happening. Because 3 hours should be sufficient time imo.